README.md

```diff
@@ -23,7 +23,7 @@ See [aces/Loris README.md](https://github.com/aces/loris) for further informatio
    sudo mkdir -p /data/$projectname/bin/mri
    sudo chown -R lorisadmin:lorisadmin /data/$projectname
    cd /data/$projectname/bin
-   git clone https://github.com/aces/Loris-MRI.git mri
+   git clone -b 16.0-dev https://github.com/aces/Loris-MRI.git mri
    ```
```

Comment:
@christinerogers 

why are we cloning the development branch? shouldn't this be 16.0?